.buildkite/commands/build-for-testing.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+gem install bundler
+bundle install
+save_cache vendor/bundle "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+
+echo "--- :cocoapods: Setting up Pods"
+
+# Caching the specs repos and global pod cache can dramatically improve Pod times
+restore_cache "$BUILDKITE_PIPELINE_SLUG-specs-repos"
+restore_cache "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
+
+restore_cache "$(hash_file Podfile.lock)"
+bundle exec pod install || bundle exec pod install --repo-update --verbose
+save_cache Pods "$(hash_file Podfile.lock)"
+
+save_cache ~/.cocoapods "$BUILDKITE_PIPELINE_SLUG-specs-repo"
+save_cache ~/Library/Caches/CocoaPods/ "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
```

Comment:
Nitpick to consider as you keep working on this automation. Both cache names are repeated above when calling `restore_cache`. Might be handy to extract them in their own variables for reuse.

Reply:
Love that we (you, I didn't do anything) already built a plugin! We should write a good https://mobile.blog post once the migration is completed sharing all the things we've done and learned.

_I also wish English had a way to distinguish between "you Jeremy" and "you folks that worked on the mobile CI project"._